README

```diff
@@ -106,6 +111,13 @@ REQUIREMENTS:
         Required for signed Verity images support:
           CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG
 
+        Required for RestrictFileSystems= in service units:
+          CONFIG_BPF
+          CONFIG_BPF_SYSCALL
+          CONFIG_BPF_LSM
+          CONFIG_DEBUG_INFO_BTF
+          CONFIG_LSM="...,bpf"
```

Comment:
This can be overwritten by the `lsm=` kernel boot parameter. Maybe the following makes sense? 

```suggestion
          CONFIG_LSM="...,bpf" or boot the kernel with lsm="...,bpf". 
```